scripts/package

```diff
@@ -11,19 +11,6 @@ cp bin/* dist/artifacts
 IMAGE=${REPO}/support-bundle-kit:${TAG}
 DOCKERFILE=package/Dockerfile
 
-docker run --privileged --rm tonistiigi/binfmt --install all
-buildx create --platform linux/amd64,linux/arm64 --use
-buildx ls
-
-# In old docker version, it doesn't support multiple values in --platform with --load.
-# So we only load image with current platform.
-# Ref: https://github.com/docker/buildx/issues/59#issuecomment-616050491
-
-# build
-buildx build --platform linux/amd64,linux/arm64 \
-  -f ${DOCKERFILE} -t ${IMAGE} .
-
-# test
```

Comment:
we already built it on, so we could remove it for local development https://github.com/rancher/support-bundle-kit/blob/8a2158dcfddd1980b98c5ad8337851a3a1b0e4f2/.github/workflows/template-build.yml#L45-L55

Reply:
But in this way, `make ci` doesn't run the packaging path. Does this mean the developers need to run local github action to build an local image for testing?

Reply:
@bk201  I'm not sure if you probably miss some codes or not. In this scripts/package, it still generates the image in the end. 

https://github.com/rancher/support-bundle-kit/blob/920b34eb8810f30fd1f9d987d4b61c6bf08d2103/scripts/package#L8-L17

BTW, they have two choices to generate image.

1. Run `make ci`, it eventually run scripts/package to build image
2. Run scripts/ci to build image

First one will cost too much time. Second one is faster, however you need to install buildx in local to use method 2.

Reply:
This answered my questions, thanks!